FaderPlugin/Windows/Config/ConfigWindow.Settings.cs

```diff
@@ -177,160 +275,186 @@ private void Settings()
                 }
             }
         }
+        #endregion
 
-        ImGui.SetCursorPos(startPos with {X = startPos.X + childSize});
+        #region Right Child : Element Configuration
+        ImGui.SetCursorPos(startPos with { X = startPos.X + childSize });
         using (var contentChild = ImRaii.Child("ConfigPage", Vector2.Zero, true))
         {
-            if (contentChild.Success)
+            if (!contentChild.Success)
+                return;
```

Comment:
if you want to early return here, do it outside the child, just change the `using ()` into a `using var xyz = ...`